server/src/main.rs

```diff
@@ -134,7 +144,7 @@ async fn main() -> Result<(), Error> {
             likes: usize,
             text: String,
             hidden: bool,
-            answered: bool,
+            answered: Option<usize>,
```

Comment:
This doesn't seem right — `server/src/test.json` is using the original live-ask.com data, which only has a boolean for `answered`.

Reply:
I didn't realize where the test data was coming from, and that you wanted to keep it as is.  I'll revert it back to the bool version.

When it comes to the `answered` field column, would you rather use a different column, or just remap to the `answered: true` values coming from `test.json` to a timestamp on the fly.

Reply:
I reverted the `test.json` file and updated the code in main to generate the `timestamp` while processing the items in the file (ie: if `answered = true` then add a `answered` timestamp for the question.